### PR TITLE
DSE-30067: Transfer ownership of project

### PR DESCRIPTION
**Pull request by navinag1989**
Added capability to transfer ownership of project during import command.
Introduced new field target_owner in import-config.ini. This field is mandatory, if user doesn't want to transfer ownership he can keep target_owner field same as username.

### CODE REVIEW

File: cmlutils/project_entrypoint.py [hunk 1]
```diff
@@ -321,6 +325,8 @@ def project_import_cmd(project_name):
         end_time = time.time()
         import_file = log_filedir + constants.IMPORT_METRIC_FILE
         write_json_file(file_path=import_file, json_data=import_data)
+        if target_owner and target_owner != username:
```
Comment (pratapdesai) [inline cmlutils/project_entrypoint.py:328]: Please add logs  indicating transfer of ownership to make it more verbose.

File: cmlutils/projects.py [hunk 2]
```diff
@@ -1721,3 +1723,23 @@ def collect_import_application_list(self, project_id):
...
+            }
+            logging.info("Transfering the ownership of project to: %s", target_owner)
+            response = call_api_v2(
+                    host=self.host,
+                    endpoint=endpoint,
+                    method="PATCH",
+                    user_token=self.apiv2_key,
+                    json_data=json_data,
+                    ca_path=self.ca_path,
+                )
```
Comment (pratapdesai) [inline cmlutils/projects.py:1744]: Also, if anything fails in between - can it be logged

**Review by pratapdesai — DISMISSED** [review]

**Comment by pratapdesai** [comment]
Also, once the change is merged - We should be updating wiki about this addendum post the release-version @navinag1989

File: cmlutils/project_entrypoint.py (hunk 1 above)
Comment (navinag1989) [reply cmlutils/project_entrypoint.py:328]: Log is present inside function - transfer_ownership

File: cmlutils/projects.py (hunk 2 above)
Comment (navinag1989) [reply cmlutils/projects.py:1744]: If the API call fails error is printed inside "call_api_v2" function. I have tested this scenario as well.

Comment (pratapdesai) [reply cmlutils/projects.py:1745]: does it print endpoint too ?

**Review by pratapdesai — APPROVED** [review]

File: cmlutils/projects.py (hunk 2 above)
Comment (navinag1989) [reply cmlutils/projects.py:1745]: Yes

**Pull request merged**